back/spec/uploaders/base_image_uploader_spec.rb

```diff
@@ -28,7 +29,83 @@
       uploader.store!(file)
 
       image = MiniMagick::Image.open(uploader.file.path)
-      expect(image.exif).to be_empty
+
+      allowed_exif_keys = %w[
+        Orientation
+        ResolutionUnit
+        XResolution
+        YCbCrPositioning
+        YResolution
+      ]
+      expect(image.exif.keys - allowed_exif_keys).to be_empty
+
+      exiftool_output = `exiftool #{Shellwords.escape(uploader.file.path)}`
```

Comment:
`Open3.capture3` could be a better option here? See https://docs.ruby-lang.org/en/master/Open3.html#method-c-capture3.